openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/Teams.spec.ts

```diff
@@ -562,4 +562,77 @@ test.describe('Teams Page', () => {
       await afterAction();
     }
   });
+
+  test('Delete a user from the table', async ({ page }) => {
+    const { apiContext, afterAction } = await getApiContext(page);
+    const id = uuid();
+
+    const table1 = new TableClass();
+
+    const team1 = new TeamClass({
+      name: `pw%percent-${id}`,
+      displayName: `pw team percent ${id}`,
+      description: 'playwright team with percent description',
+      teamType: 'Group',
+    });
+
+    await table1.create(apiContext);
```

Comment:
Not sure if we require this, as we are not performing any test around the Entity in this step. Can you verify and remove it if not needed ?

Reply:
We require this to create a new team and locate Users tab in that teams page.